mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -1271,3 +1331,61 @@ def _get_orderby_clauses(order_by_list, session):
         clauses.append(SqlRun.start_time.desc())
     clauses.append(SqlRun.run_uuid)
     return select_clauses, clauses, ordering_joins
+
+
+def _get_search_experiments_filter_clauses(parsed_filters, dialect):
+    attribute_filters = []
+    non_attribute_filters = []
+    for f in parsed_filters:
+        typ = f["type"]
+        key = f["key"]
+        comparator = f["comparator"]
+        value = f["value"]
+        if typ == "attribute":
+            attr = getattr(SqlExperiment, key)
+            if comparator in ("LIKE", "ILIKE"):
+                f = SearchUtils.get_sql_filter_ops(attr, comparator, dialect)(value)
+            elif comparator == "=":
+                f = attr == value
+            elif comparator == "!=":
+                f = attr != value
+            else:
+                raise MlflowException.invalid_parameter_value(
+                    f"Invalid comparator for attribute: {comparator}"
+                )
+            attribute_filters.append(f)
+        elif typ == "tag":
```

Comment:
Can we support `LIKE` / `ILIKE` comparators for tag-based search as well?

Reply:
Sure!

Reply:
Added support for `LIKE` and `ILIKE`.